brainstormforce/astra
# AST-1596 - Match up WP block editor & their respective frontend UI

### Description
- Filter to use to test new UI
```
add_filter( 'astra_get_option_enable-brand-new-editor-experience', '__return_true' );
```
- Sidebar setting's controller update from dropdown to image picker - https://share.bsf.io/JruGKQNm - https://share.bsf.io/Z4u1WEJd
- "Disable Page Title" toggle setting removed and managed easily near title input - https://share.bsf.io/NQuxy0Ek
- All old CSS removed we added tilldate (except required ones like easy interactive hover box-shadow, spacer BG etc) 
- Users can override sizes by following ways -
- 1. Either by CSS (overriding by vars)
- 2. by filters provided
```
// But this will only update values on frontend not in editor mode.
:root body {
    --ast-content-width-size: 910px;
    --ast-wide-width-size: 1200px;
}
```
- Filters to update sizes - 
```
function astra_update_block_content_size( $content_size ) {
	return '840px'; // Default 910px overriden by 840px.
}

function astra_update_block_wide_size( $wide_size ) {
	return '1000px'; // Default 1200px overriden by 1000px.
}

add_filter( 'astra_block_content_width', 'astra_update_block_content_size' );
add_filter( 'astra_block_wide_width', 'astra_update_block_wide_size' );
```

### Screenshots


### Types of changes
- New feature (non-breaking change )

### How has this been tested?
- Check as per plan & card comments - https://wp-astra.atlassian.net/browse/AST-1596
- https://brainstormforce.slack.com/archives/C0FB28KK3/p1645688114969249
- Test every block of WP
- Check block layouts
- Check site & Astra layouts
- Check by importing layout if anything breaking

### Checklist:
